src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -23,15 +23,18 @@ internal interface IOktaApi {
 internal class OktaApi : IOktaApi {
 	private readonly CookieContainer _cookieContainer;
 	private readonly HttpClient _httpClient;
+	private string organization = string.Empty;
```

Comment:
Please don't initialize things to invalid non-null values just so we can mark things as non-nullable and skip null checks.
Nullables are good. Forced null checks are good. That's the compiler helping us avoid null errors.
False non-null values are much worse.

also - we name private fields with an underscore
```suggestion
	private string? _organization;
```

Reply:
Yea that's fair. I saw we couldn't reach the exception without having that value set already but I can see the reasoning

Reply:
related - https://desire2learn.atlassian.net/browse/VUL-104 this should help